python-stdlib/pprint/pprint.py

```diff
@@ -1,6 +1,57 @@
-def pformat(obj):
-    return repr(obj)
+from sys import stdout as _stdout
 
+def pformat(obj, indent=1, sort_dicts=True):
+    'Simply calls pprint() but capturing prints into a string and returning'
```

Comment:
docstrings style should be like `"""Simply calls pprint()"""` or if multiline

```
"""Short description.

Long description here.
"""
```

Reply:
Done.